src/app/header/product/product.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+  <div class="row">
+    <div class="col-2">
+      <nav class="nav flex-column">
+        <a class="nav-link active" aria-current="page">
+          <button (click)="getProductByCategory('Fruits & Vegetables')">
+            Fruits & Vegetables
+          </button></a
+        >
+
+        <a class="nav-link">
+          <button (click)="getProductByCategory('Bakery Cakes and Dairy')">
+            Bakery Cakes and Dairy
+          </button>
+        </a>
+        <a class="nav-link">
+          <button (click)="getProductByCategory('Beverages')">
+            Beverages
+          </button></a
+        >
+        <a class="nav-link">
+          <button (click)="getProductByCategory('Beauty and Hygiene')">
+            Beauty and Hygiene
+          </button></a
+        >
+        <a class="nav-link">
+          <button (click)="getProductByCategory('Baby Care')">
+            Baby Care
+          </button></a
+        >
+      </nav>
+    </div>
+    <div class="col-10">
+      <div class="row" *ngFor="let row of productPerRow">
```

Comment:
Please separate categories from the product listing, and product listing from the actual products

Reply:
done